src/CMake/dkms-aws.cmake

```diff
@@ -16,19 +16,19 @@ set(DKMS_POSTINST "postinst-aws")
 set(DKMS_PRERM "prerm-aws")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`